backend/core/src/seeds/listings/listing-detroit-10157.ts

```diff
@@ -177,6 +178,34 @@ export class Listing10157Seed extends ListingDefaultSeed {
       preferences: [],
     }
 
+    const nccUnitsSummaryToBeCreated: UnitsSummaryCreateDto[] = []
```

Comment:
For this set of properties, could you go to the rentlinx site and put in the correct monthlyRent ranges? Also, does our units summary view have sq footage? That information is also in the rentlinx site.

Reply:
There is an optional sq footage min and max. I can add it for this one.